File: bottlesOfBeer.py
bottle = 99

while(bottle > 1):
    if bottle > 2:
        print("{} bottles of beers on the wall {} bottles of beer, take one down pass it around {} bottles of beer on the wall".format(bottle, bottle, bottle-1))
    else:
        print("{} bottles of beers on the wall {} bottles of beer, take one down pass it around {} bottle of beer on the wall".format(bottle, bottle, bottle-1))
    bottle = bottle-1

File: bottlesOfBeer/bottlesOfBeer.java
package com.beginner_project_solutions;

public class Main {

    public static void main(String[] args) {
        short bottles = 99;

        while(bottles > 1){
            if(bottles > 1) {
                System.out.println(bottles + " bottles of beer on the wall " + bottles + " bottles of beer");
                System.out.println("Take one down pass it around " + (bottles - 1) + " bottles of beer on the wall");
            }
            else{
                System.out.println(bottles + " bottles of beer on the wall " + bottles + " bottles of beer");
                System.out.println("Take one down pass it around " + (bottles - 1) + " bottle of beer on the wall");
            }
            bottles--;
        }
    }
}
